aap_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -274,7 +278,7 @@ export const AnsibleChatbot: React.FunctionComponent<ChatbotContext> = (
                       </div>
                     ),
                   )}
-                  {isLoading ? (
+                  {messages.at(-1)?.role === "user" && isLoading ? (
```

Comment:
There is a small time period when `isLoading` is set and a bot message is already added to the last of messages list. In such a case, we do not need to the loading animation icon.